app/scripts/controllers/permissions/enums.ts

```diff
@@ -2,4 +2,5 @@ export enum NOTIFICATION_NAMES {
   accountsChanged = 'metamask_accountsChanged',
   unlockStateChanged = 'metamask_unlockStateChanged',
   chainChanged = 'metamask_chainChanged',
+  sessionChanged = 'wallet_sessionChanged',
```

Comment:
Is this needed? I can't see any references to it

Reply:
not needed here. will move into the Multichain PR

Reply:
https://github.com/MetaMask/metamask-extension/pull/27847/commits/0fead5c0c5229f2387daf49a4e961336944cac99